components/organisms/PhaseBanner.js

```diff
@@ -0,0 +1,297 @@
+        // handle error if fetch fails
+        // fetch only fails if there is no internet connection not for the actual
+        // request so there is nothing really to do here other than to log it
+        console.log(e);
+      });
+
+      setSubmitted(true);
+    } else {
+      setFeedbackError(error.message);
+    }
+  };
+
+  //Function for changing feedback button state
+  function onfeedbackClick() {
+    const feedbackDropdown = document.getElementById("feedbackDropdown");
+
+    feedbackDropdown.classList.toggle("active");
+    ariaExpanded === true ? setAriaExpanded(false) : setAriaExpanded(true);
+
+    ariaExpanded === false ? setCaretHtml("&#9660;") : setCaretHtml("&#9656;");
+  }
+
+  return (
+    <>
+      <div className="bg-circle-color">
+        <div className="block sm:flex py-4 layout-container">
+          <div
+            className={`flex justify-between sm:block sm:w-max ${
+              feedbackActive ? "mt-2" : ""
+            }`}
+          >
+            <span className="font-body text-xs text-circle-color font-extrabold block w-max bg-white px-4 py-1 my-auto leading-6">
+              {phase}
+            </span>
+            {feedbackActive ? (
+              <ActionButton
+                id="back-projects"
+                dataCy="back-projects"
+                dataTestId="back-projects"
+                custom="font-body text-xs mt-0 sm:mt-4 underline hover:text-canada-footer-hover-font-blue text-white block w-32"
+                text={t("backProjects")}
+                href={t("breadCrumbsHref2")}
+              />
+            ) : (
+              ""
+            )}
+          </div>
+          <div>
+            <p className="font-body text-xs lg:text-sm mt-5 sm:mt-auto text-white sm:ml-4 pt-1 my-auto sm:mb-0 lg:pb-1">
+              {children}
+            </p>
+            {feedbackActive ? (
+              <button
+                id="feedbackButton"
+                onClick={onfeedbackClick}
+                className="bg-circle-color font-body text-xs lg:text-sm text-white flex text-left sm:ml-4 my-2 sm:my-0"
+                aria-haspopup="true"
+                aria-expanded={ariaExpanded}
+                aria-controls="feedbackDropdown"
+                data-testid="feedbackButton"
+              >
+                <span
+                  id="caret"
+                  className={`${
+                    caretHtml === "&#9660;" ? "text-sm" : "text-p"
+                  } leading-7 lg:leading-5`}
+                  dangerouslySetInnerHTML={{ __html: caretHtml }}
+                />
+                <strong className="underline mt-2 lg:mt-0 ml-2">
+                  {t("giveFeedback")}
+                </strong>
+              </button>
+            ) : (
+              ""
+            )}
+          </div>
+        </div>
+      </div>
+      <div
+        id="feedbackDropdown"
+        className="feedbackDropdown bg-custom-blue-blue"
+        data-testid="feedbackDropdown"
+      >
```

Comment:
This `div` would be wrapped by `showFeedback`,
```
{showFeedback ? (
        <div
          id="feedbackDropdown"
          className="bg-custom-blue-blue"
          data-testid="feedbackDropdown"
        >
...
</div>
: ""}
```
I'm not sure if this div should always be rendered or not since you're using `aria-control` to specify that the button will open or close this div.